src/components/Portfolio/PortfolioHoldings/PortfolioHoldingRow.tsx

```diff
@@ -60,73 +64,117 @@ export default function PortfolioHoldingRow({
 
   return (
     <>
-      <HoldingTableRow>
-        <HoldingTableCell>
+      <HoldingTableRow
+        tabIndex={-1}
+        selected={isItemSelected}
+        aria-checked={isItemSelected}
+        onClick={(event) => handleClick(event, tickerSymbol)}>
+        <HoldingTableCell
+          style={{
+            width: "40px",
+            borderRadius: "8px 0 0 8px",
+            padding: "0 8px 0 16px",
+          }}>
           <IconButton
+            style={{
+              width: "16px",
+              height: "16px",
+            }}
             aria-label="expand row"
             size="small"
             onClick={onExpandRowClick}>
-            {isRowOpen ? <KeyboardArrowDownIcon /> : <KeyboardArrowRightIcon />}
+            {isRowOpen ? (
+              <Icon
+                icon="chevron-down"
+                size={16}
+                variant="tertiary"
+                disabled={false}
+              />
+            ) : (
+              <Icon
+                icon="chevron-right"
+                size={16}
+                variant="tertiary"
+                disabled={false}
+              />
+            )}
```

Comment:
```suggestion
              <Icon
                icon=`${isRowOpen ? "chevron-down" : "chevron-right"`
                size={16}
                variant="tertiary"
                disabled={false}
              />
```

바뀌는 부분만 삼항연산자로 묶어도 될 것 같네요

Reply:
SO GOOD 👍